tests/unit/conftest.py

```diff
@@ -58,11 +59,12 @@ def __init__(self, config, success_channel, error_channel, **kwargs):
 
 @zope.interface.implementer(interfaces.IEnricherClient)
 class EnricherStub:
-    def __init__(self, config, success_channel, error_channel, **kwargs):
+    def __init__(self, config, success_channel, error_channel, metrics=None):
```

Comment:
Should these still be `None`? Since we are going to provide plugins w/ a metrics object. Or actually, perhaps this part is updated in #23 ?

Reply:
Yeah, that's reasonable. Although I guess is how an implementer would do it if they don't want to care about metrics, right?

Reply:
Actually, @econchick I'll wait for your response to that ^? Since a lot of tests that don't care about metrics just broke when when I updated that, I want to make sure it's important before I go through to fix everything.

Reply:
Ah sorry I didn't see this earlier. I think you're right; plugins don't necessarily need to care about metrics, but they will get an object. So `metrics=None` is indeed correct, my bad.